client/consensus/Cargo.toml

```diff
@@ -9,18 +9,18 @@ repository = "https://github.com/paritytech/frontier/"
 
 [dependencies]
 codec = { package = "parity-scale-codec", version = "1.3.4", features = ["derive"] }
-sp-core = { version = "2.0.0", git = "https://github.com/paritytech/substrate.git", branch = "frontier" }
-sp-blockchain = { version = "2.0.0", git = "https://github.com/paritytech/substrate.git", branch = "frontier" }
-sp-runtime = { version = "2.0.0", git = "https://github.com/paritytech/substrate.git", branch = "frontier" }
-sp-api = { version = "2.0.0", git = "https://github.com/paritytech/substrate.git", branch = "frontier" }
-sc-client-api = { version = "2.0.0", git = "https://github.com/paritytech/substrate.git", branch = "frontier" }
-sp-block-builder = { version = "2.0.0", git = "https://github.com/paritytech/substrate.git", branch = "frontier" }
-sp-inherents = { version = "2.0.0", git = "https://github.com/paritytech/substrate.git", branch = "frontier" }
+sp-core = { version = "2.0.0", git = "https://github.com/paritytech/substrate.git", branch = "master" }
```

Comment:
Would you mind if we revert the branch change? I updated the `frontier` branch to commit `39278096356942a087213b0f21c976ac100cb8f8`.